gpt4all-chat/chatgpt.h

```diff
@@ -24,6 +24,7 @@ class ChatGPT : public QObject, public LLModel {
         std::function<bool(int32_t, const std::string&)> responseCallback,
         std::function<bool(bool)> recalculateCallback,
         PromptContext &ctx) override;
+    bool evalTokens(PromptContext &ctx, const std::vector<int32_t> &tokens) override { return true; }
```

Comment:
This function just returning `true` if not implemented may be debatable, but it is required for the ChatGPT implementation in the frontend.